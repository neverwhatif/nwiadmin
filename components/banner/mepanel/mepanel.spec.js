import React from 'react';
import { shallow } from 'enzyme';

import MePanel from './index';

const data = {
    name: 'Lorem Ipsum',
};

describe('MePanel', () => {
    const component = shallow(<MePanel data={data} />);

    it('should render', () => {
        expect(component).toMatchSnapshot();
    });
    it('should render name', () => {
        expect(component.text()).toBe('Lorem Ipsum');
    });
});
