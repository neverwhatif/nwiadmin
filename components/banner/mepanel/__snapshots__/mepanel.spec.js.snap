// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MePanel should render 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <MePanel
    data={
      Object {
        "name": "Lorem Ipsum",
      }
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2
          className="title"
        >
          Lorem Ipsum
        </h2>,
        <button
          onClick={[Function]}
        >
          Log out
        </button>,
      ],
      "className": "root",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Lorem Ipsum",
          "className": "title",
        },
        "ref": null,
        "rendered": "Lorem Ipsum",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Log out",
          "onClick": [Function],
        },
        "ref": null,
        "rendered": "Log out",
        "type": "button",
      },
    ],
    "type": "aside",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2
            className="title"
          >
            Lorem Ipsum
          </h2>,
          <button
            onClick={[Function]}
          >
            Log out
          </button>,
        ],
        "className": "root",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Lorem Ipsum",
            "className": "title",
          },
          "ref": null,
          "rendered": "Lorem Ipsum",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Log out",
            "onClick": [Function],
          },
          "ref": null,
          "rendered": "Log out",
          "type": "button",
        },
      ],
      "type": "aside",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
